cfdtools/meshbase/_mesh.py

```diff
@@ -382,16 +367,13 @@ def reindex_boundaryfaces(self):
             fdict['index'] = _conn.indexlist(ilist=newindex[fdict['index'].list()].tolist())
             # fdict['index'].compress() # not expected
         if 'face2cell' in self._faces['boundary']:
-            self._faces['boundary']['face2cell'].conn = (
-                self._faces['boundary']['face2cell'].conn[oldindex, :]
-            )
+            self._faces['boundary']['face2cell'].conn = self._faces['boundary']['face2cell'].conn[oldindex, :]
 
     def printinfo(self, detailed=False):
         api.io.printstd(f"nnode: {self.nnode}")
         for c in ('x', 'y', 'z'):
             api.io.printstd(
-                f"  {c} min:avg:max ="
-                " {:.3f}:{:.3f}:{:.3f}".format(*minavgmax(self._nodes[c])),
+                f"  {c} min:avg:max =" " {:.3f}:{:.3f}:{:.3f}".format(*minavgmax(self._nodes[c])),
```

Comment:
f"  {c} min:avg:max = {:.3f}:{:.3f}:{:.3f}".format(*minavgmax(self._nodes[c])),

Reply:
cannot mix f" " and " ".format